index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://use.fontawesome.com/bba5c6ae87.js"></script>
+    <script src="https://use.fontawesome.com/83ac61cd0f.js"></script>
+    <title>My Portfolio</title>
+</head>
+<body>
+<header class="wrapper">
+    <nav class="flex-nav">
+        <ul>
+            <li ><a class="logo" href="#">Chidiebere</a> </li>
+            <li class="menu-icon toggle-icon"><i class="fa fa-bars" aria-hidden="true"></i></li>
+        </ul>
+    </nav>
+</header>
+
+
+<section>
+    <div class="headline">
+
+        <h1 class="intro name">Hey There. I'm Chidiebere Ojingwa</h1>
+        <h2 class="intro profession">I'am a Software Developer</h2>
+        <p class="intro task"> I can help you build a product, feature or <br> website
+        look through some of my work <br>and experience! If you like what you see <br> and
+        have a project you need, <br> don't hesitate to contact me</p>
+
+        <ul class="social">
+            <li><a href="#" class="fa fa-github"></a></li>
+            <li><a href="#" class="fa fa-linkedin"></a></li>
+            <li><a href="#" class="fa fa-angellist"></a></li>
+            <li><a href="#" class="fa fa-twitter"></a></li>
+            <li><a href="#" class="fa fa-medium"></a></li>
+          </ul>
+    </div>
+
+</section>
```

Comment:
Original design 
![image](https://user-images.githubusercontent.com/43698511/138174794-e11b75f8-f6fb-42f6-944d-516e4960b0c5.png)
Your webpage :
![image](https://user-images.githubusercontent.com/43698511/138174844-95cc0359-1868-4386-977a-33c87af7c0c5.png)

- [ ] [OPTIONAL] Since the first reviewer didn't mention it. I marked it as optional but I HIGHLY recommend you to do the changes before merging. Following the original design is one of the project's requirements as it is an important skill for a Front end dev.

As you can see from the pictures. Your webpage looks a bit different from the original design.

-  Kindly adjust the size of the icons to match the original design
- Kindly adjust the distance between the elements of the sections.
- Kindly modify the font family and font size of the text to match the original design.